django/contrib/admin/options.py

```diff
@@ -1055,13 +1055,16 @@ def render_change_form(self, request, context, add=False, change=False, form_url
         preserved_filters = self.get_preserved_filters(request)
         form_url = add_preserved_filters({'preserved_filters': preserved_filters, 'opts': opts}, form_url)
         view_on_site_url = self.get_view_on_site_url(obj)
+        is_multipart = context['adminform'].form.is_multipart() or any(
+            fset.formset.form().is_multipart() for fset in context['inline_admin_formsets']
+        )
         context.update({
             'add': add,
             'change': change,
             'has_add_permission': self.has_add_permission(request),
             'has_change_permission': self.has_change_permission(request, obj),
             'has_delete_permission': self.has_delete_permission(request, obj),
-            'has_file_field': True,  # FIXME - this should check if form or formsets have a FileField,
```

Comment:
If you want to use a new name, that's okay with me, but I think `'has_file_field'` should remain for backwards compatibility.

Reply:
Okay, then I think you could skip the `is_multipart` variable and put the calculation directly in the dict.